apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -837,24 +776,14 @@ async fn ingest_route(
         .find(|(k, _)| k.to_str().unwrap_or("").to_lowercase().eq(&route_str));
 
     match matching_route {
-        Some((_, route_meta)) => match route_meta.format {
-            EndpointIngestionFormat::Json => Ok(handle_json_req(
-                &configured_producer,
-                &route_meta.kafka_topic_name,
-                &route_meta.data_model,
-                req,
-                &jwt_config,
-            )
-            .await),
-            EndpointIngestionFormat::JsonArray => Ok(handle_json_array_body(
-                &configured_producer,
-                &route_meta.kafka_topic_name,
-                &route_meta.data_model,
-                req,
-                &jwt_config,
-            )
-            .await),
-        },
+        Some((_, route_meta)) => Ok(handle_json_array_body(
```

Comment:
Does this handle for arrays and single objects?

From the code that looks surprising